ota-plus-web/app/reactapp/src/containers/Device.jsx

```diff
@@ -44,23 +44,14 @@ class Device extends Component {
     const { stores } = this.props;
     const { devicesStore } = stores;
     const deviceId = devicesStore.device.uuid;
-    const data = {
-      correlationId,
-      device: deviceId,
-    };
-    devicesStore.cancelMtuUpdate(data);
+    devicesStore.cancelMtuUpdate(deviceId);
```

Comment:
Please remove correlationId from input parameters of cancelMtuUpdate function.

Reply:
Right! Removed.